tests/publiccloud/boottime.pm

```diff
@@ -99,9 +99,9 @@ sub run {
     my $thresholds = {
         # First boot after provisioning
         kernel     => 15,
-        userspace  => 60,
-        initrd     => 10,
-        overall    => 120,
+        userspace  => 180,
```

Comment:
180 seconds for userspace seems a bit too steep. The tests were failing recently due to boot time being something like 65 seconds.
Generally, boot time is a very good thing to keep an eye on, we are not yet sure what exactly caused the slowdown in the most recent images.

I would suggest setting it to something like 90 seconds for now.

Reply:
done